backend/app/views/spree/admin/shared/_navigation_header.html.erb

```diff
@@ -1,5 +1,5 @@
 <header class="admin-nav-header">
-  <%= link_to spree.admin_path, class: 'brand-link' do %>
```

Comment:
Shops may have used this class for deface overrides. We should not remove it now.